src/CommentsDrawerLink.jsx

```diff
@@ -1,94 +1,123 @@
+            </button>
+        </div>;
     }
 
-    return (
-      <VisibilitySensor
-        onChange={(visible) => {
-          if (visible && this.state.isVisible !== visible) {
-            this.setState({ isVisible: visible })
-          }
-        }}
-        partialVisibility
-        scrollCheck>
+    defaultHeader() {
+        return <Trans>header_comments</Trans>;
+    }
+
+    render() {
+        let comments;
+        if ( this.state.isVisible ) {
+            comments = <Comments {...this.props} commentCountRefreshed={this.updateCommentCount.bind(this)}/>
+        }
+
+        return (
+            <VisibilitySensor
+                onChange={(visible) => {
+                    if ( visible && this.state.isVisible !== visible ) {
+                        this.setState({isVisible: visible})
+                    }
+                }}
+                partialVisibility
+                scrollCheck>
         <span>
           <div className="comment-drawer-button">
             <span className="fa fa-comments-o"
-              onClick={() => this.setState({
-                commentsDrawerOpen: true
-              })}/>
-            <span key="test" className="comment-count-badge" style={{opacity: this.state.opacity}}>{this.state.availableComments}</span>
+                  onClick={() => this.setState({
+                      commentsDrawerOpen: true
+                  })}/>
+            <span key="test" className="comment-count-badge"
+                  style={{opacity: this.state.opacity}}>{this.state.availableComments}</span>
           </div>
           <Drawer
-            show={this.state.commentsDrawerOpen}
-            onRequestHide={() => this.setState({commentsDrawerOpen: false})}
-            header={this.props.header ? this.props.header : 'Comments'}
-            position={this.props.position === 'left' ? 'left' : 'right'}
-            closeOnClickOutside={true}
-            footer={this.props.footer ? this.props.footer : footer}>
+              show={this.state.commentsDrawerOpen}
+              onRequestHide={() => this.setState({commentsDrawerOpen: false})}
+              header={this.props.header || this.defaultHeader()}
+              position={this.props.position}
+              closeOnClickOutside={true}
+              footer={this.props.footer || this.defaultFooter()}>
             {comments}
           </Drawer>
         </span>
-      </ VisibilitySensor>
-    );
-  }
+            </ VisibilitySensor>
+        );
+    }
 }
 
 _CommentsDrawerLink.propTypes = {
-  accessToken: PropTypes.string.isRequired,
-  resourceUri: PropTypes.string.isRequired,
-  newestFirst: PropTypes.bool,
-  editComments: PropTypes.bool,
-  refreshInterval: PropTypes.number,
-  position: PropTypes.string,
-  header: PropTypes.node,
-  footer: PropTypes.node,
-  opened: PropTypes.bool
-};
\ No newline at end of file
+    locale: PropTypes.string,
```

Comment:
New attribute that defines the language (all components). For now, it accepts only the 3-letter code but this feels odd so we might want to change it.